pyspecProcScripts/third_level/process_enhancement.py

```diff
@@ -24,11 +25,6 @@
 logger = init_logging("info")
 t2 = symbols('t2')
 thesecolors = cycle(list('bgrcmykw'))
-def select_pathway(s,pathway):
-    retval = s
-    for k,v in pathway.items():
-        retval = retval[k,v]
-    return retval
```

Comment:
while I was at it, I searched for all instances of `def select_pathway` and imported instead.